components/css/partials/_accordion.scss

```diff
@@ -32,5 +41,29 @@
 				content:"\f0fe";
 			}
 		}
+		@each $color,$pair in $colorPairs-accessible{
+			&-#{$color}{
+				background-color:nth($pair,1);
+				color:nth($pair,2);
+			}
+		}
+	}
+	&_list{
```

Comment:
What if the content is not a list?

Reply:
I suppose that's fair. I was trying to consider what other elements could be placed in the accordion. With the current design, we only account for having a list of items. Not to say that it has to be a `<ul>` or `<ol>` but it is just some items grouped together. I suppose this could be changed to **content** but I didn't want to confuse it with **item** element.